tests/test_grid.py

```diff
@@ -7,7 +7,9 @@
 import gemmi
```

Comment:
A lot of the changes in this file are formatting changes - do we have auto formatting in this repo?

Reply:
nope